packages/components/src/internal/components/user/UsersGridPanel.tsx

```diff
@@ -343,6 +343,7 @@ export class UsersGridPanelImpl extends PureComponent<Props, State> {
                                 title={capitalizeFirstChar(usersView) + ' Users'}
                                 ButtonsComponent={() => this.renderButtons()}
                                 highlightLastSelectedRow
+                                allowViewCustomization={false}
```

Comment:
I'm going back and forth on this one.  Is here a reason we really shouldn't allow customization here? Only admins will ever see it. Of course, there's no a lot that's interesting in the users domain, so we can leave it uncustomizable and enable it if there's interest.